snooty.toml

```diff
@@ -18,9 +18,9 @@ sharedinclude_root = "https://raw.githubusercontent.com/10gen/docs-shared/main/"
 driver = "java"
 driver-long = "MongoDB Java Driver"
 version = "4.7"
-full-version = "{+version+}.0"
+full-version = "{+version+}.1"
 
 package-name-org = "mongodb-org"
 api = "https://mongodb.github.io/mongo-java-driver/{+version+}"
 stable-api = "Stable API"
-mongocrypt-version = "1.5.1.1"                                  # ensure compatibilty with driver version
+mongocrypt-version = "1.5.2"                                  # ensure compatibilty with driver version
```

Comment:
Can you remove the comment and instead add this to the snooty.toml updates needed for docs-java in the [Task Checklist](https://docs.google.com/spreadsheets/d/11oY2n5dCsfP5iNCf9L8j3-YiE6ESpjq6TmLdBeaNdN4/edit#gid=0) ? The comment doesn't seem clear whether it is stating that there's uncertainty of the current compatibility state with the [current] driver version.

Reply:
Good point, I'll remove the comment.

I believe the "snooty.toml" entry in the Java driver tab of the task checklist is sufficient (check that links are still working), so not sure if anything needs to be added.

Reply:
I think this might involve checking whether a runnable project still works? I don't think this constant is used in a link.